packages/admin-portal/src/components/pages/login.tsx

```diff
@@ -1,98 +1,31 @@
 import * as React from 'react'
-import { useLocation, useRouteMatch } from 'react-router'
-import { Link, Redirect } from 'react-router-dom'
-import { Dispatch } from 'redux'
-import { useSelector, useDispatch } from 'react-redux'
-import { authChangeLoginType } from '@/actions/auth'
-import { showNotificationMessage } from '@/actions/notification-message'
-import { selectLoginSession, selectRefreshSession, selectLoginType, selectIsDesktopMode } from '@/selector/auth'
-import { LoginType, redirectToLogin } from '@reapit/cognito-auth'
+import { Redirect } from 'react-router-dom'
+import { useSelector } from 'react-redux'
+import { selectLoginSession, selectRefreshSession } from '@/selector/auth'
+import { redirectToLogin } from '@reapit/cognito-auth'
 import { Button, Level, FlexContainerBasic, Section } from '@reapit/elements'
-import { getLoginTypeByPath, getDefaultPathByLoginType, getDefaultRouteByLoginType } from '@/utils/auth-route'
 import Routes from '@/constants/routes'
-import messages from '@/constants/messages'
-import {
-  getCookieString,
-  COOKIE_DEVELOPER_FIRST_TIME_LOGIN_COMPLETE,
-  COOKIE_CLIENT_FIRST_TIME_LOGIN_COMPLETE,
-} from '@/utils/cookie'
 import loginStyles from '@/styles/pages/login.scss?mod'
 import logoImage from '@/assets/images/reapit-graphic.jpg'
 import connectImage from '@/assets/images/reapit-connect.png'
 
-const { wrapper, container, image, register, registerLevel, loginButton } = loginStyles
+const { wrapper, container, image, registerLevel, loginButton } = loginStyles
 
 export type LoginProps = {}
 
-export const handleShowNotificationAfterPasswordChanged = (
-  isPasswordChanged: boolean,
-  localStorage: Storage,
-  dispatch: Dispatch,
-) => {
-  return () => {
-    if (isPasswordChanged) {
-      dispatch(showNotificationMessage({ variant: 'info', message: messages.PASSWORD_CHANGED_SUCCESSFULLY }))
-      localStorage.removeItem('isPasswordChanged')
-    }
-  }
-}
-
-export const handleChangeLoginType = (dispatch: Dispatch, loginType: LoginType) => {
-  return () => {
-    dispatch(authChangeLoginType(loginType))
-  }
-}
-
-export const onLoginButtonClick = (
-  loginType: LoginType,
-  isDeveloperFirstTimeLoginComplete: boolean,
-  isClientFirstTimeLoginComplete: boolean,
-) => {
-  return () => {
-    const redirectRoute = getDefaultRouteByLoginType({
-      loginType,
-      isDeveloperFirstTimeLoginComplete,
-      isClientFirstTimeLoginComplete,
-    })
-
-    redirectToLogin(window.reapit.config.cognitoClientId, redirectRoute, loginType)
-  }
+export const onLoginButtonClick = () => {
+  const redirectRoute = `${window.location.origin}${Routes.ADMIN_APPROVALS}`
```

Comment:
I think no need `ADMIN_` in prefix of routes

Reply:
@tanphamhaiduong all route name variables will be update in the next pull request. This pr just focused on login page.